lib/typescript/components/message/MessageWrapper/MessageContainer/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {ReactNode} from 'react';
+import {SourceMessage, CommandUnion, ContentType} from 'render';
+import {DeliveryState, Source, Message} from 'model';
+import {ReactComponent as ErrorMessageIcon} from 'assets/images/icons/errorMessage.svg';
+import {Reaction} from '../../Reaction';
+import styles from './index.module.scss';
+
+interface MessageContainerProps {
+  message: Message;
+  source: Source;
+  isChatPlugin: boolean;
```

Comment:
nitpick: It'd be great to get rid of this prop. Especially since it seems like the chatplugin itself can simply pass down the `invertSides` and `commandCallback` props.